recipes/icqsol/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: icqsol
+  version: "0.3.20"
+
+about:
+  home: 'https://github.com/pletzer/icqsol'
+  summary: 'A collection of utilities for constructing complex geometries from primitive shapes'
+  license: MIT
+
+source:
+  git_url: https://github.com/pletzer/icqsol.git
+  git_rev: 05b229c
+
+build:
+  number: 0
+
+test:
+  # Python imports
+  imports:
+    - csg
+    - icqsol
+    - numpy
+    - triangle
+    - vtk
+
+requirements:
+  build:
+    - cmake
```

Comment:
ok, I thought this may be required, but I was bumping into issues on Linux because I'm running gcc 6.1 and it looks like some of the required packages here (e.g., vtk) require older versions of gcc, so conda was not compiling this correctly on Linux for me.  I'll add these in though - thanks!